Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM ruby:latest
+RUN apt-get update && apt-get install -y nodejs npm
```

Comment:
Let's add this to the line that is installing needed packages.

It would be like:
```
RUN apt-get update -y && apt-get install -y --no-install-recommends \
    locales \
    nodejs npm \
    imagemagick \
    build-essential \
    zlib1g-dev \
    python3-pip \
    inotify-tools procps && \
    apt-get clean && rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/* && \
    pip install nbconvert --break-system-packages
```